services/ui-src/src/components/cards/EntityCard/EntityCard.tsx

```diff
@@ -114,15 +114,22 @@ export const EntityCard = ({
           printVersion={!!printVersion}
         />
         {openAddEditEntityModal && (
-          <Button
-            variant="outline"
-            size="sm"
-            sx={sx.editButton}
-            leftIcon={<Image src={editIcon} alt="edit icon" height="1rem" />}
-            onClick={() => openAddEditEntityModal(entity)}
-          >
-            {verbiage.editEntityButtonText}
-          </Button>
+          <>
+            {!formattedEntityData.reportingPeriod ? (
+              <Text sx={sx.unfinishedMeasureMessage}>
+                {verbiage.unfinishedMeasureMessage}
+              </Text>
+            ) : null}
```

Comment:
Question: We only want this text to display for quality measures without reporting periods, but this code is also used to display other entity types - access measures, sanctions and so on - none of which have reporting periods. So on first read, it seems like this code would incorrectly display the unfinished measure message for those entity types as well. But when I tested in Cloudfront, I didn't see that problem.

It took me a while to figure out that this code _does_ display an unfinished measure message for other entity types - except that those entities don't have any verbiage defined for that message, so we're actually only displaying an empty paragraph.

This isn't a huge issue, but there are two effects. For one, the code is a little puzzling (at least, it took me a minute). For two, that paragraph does reserve a bit of extra space on the page despite being empty, due to its margin - which could be considered a styling bug. This screenshot shows how much space we're talking about:

![image](https://github.com/Enterprise-CMCS/macpro-mdct-mcr/assets/126210497/25cd1245-5b11-45e1-a1da-fd1df3f225c6)

I don't think this is blocking, but maybe the check could be made more specific? But I'm not sure what the best way to do that would be, because this is a ticket that really tests our separation of verbiage and component functionality.

Reply:
Do we pass in the entity type at all here? If so, we could pretty easily add that as a condition for the text to show.

Reply:
I update this to be a && instead of a ternary - does that address this issue?

Reply:
ah yup, that was an oversight on my part - updated to check the entityType